renderer/components/Pay/Pay.js

```diff
@@ -738,12 +738,12 @@ class Pay extends React.Component {
                           <Text fontSize="xs" textAlign="center">
                             <CryptoValue value={walletBalanceConfirmed} />
                             {` `}
-                            {cryptoUnitName} (onchain),
+                            {cryptoUnitName} <FormattedMessage {...messages.onchain_label} />
```

Comment:
@bolatovumar I'd suggest combining these into a single translation string that includes both `cryptoUnitName` and the `onchain_label`.

So that the translation strings looks more like this:

```jsx
{amont} {cryptoUnitName} (onchain)
```

And then use like:

```jsx
<FormattedMessage {...messages.onchain_balance} values={{ amount, cryptoUnitName }} />
```
This will provide greater flexibility for languages that don't follow the latin text formatting rules.

Reply:
Makes sense. Will do.

Reply:
Done.